mlflow/spark.py

```diff
@@ -741,6 +741,56 @@ def _load_pyfunc(path):
     return _PyFuncModelWrapper(spark, _load_model(model_uri=path))
 
 
+def find_and_set_features_col_as_vector_if_needed(spark_df, spark_model):
```

Comment:
Can we make this an internal method so that it doesn't appear in the docs?

```suggestion
def _find_and_set_features_col_as_vector_if_needed(spark_df, spark_model):
```